### PR TITLE
WIP jenkins less

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] Client test.sh
- [x] Go test.sh
- [ ] change_target can be null, force all tests

### CODE REVIEW

**Comment by chrisnojima** [comment]
- [ ] need correct merge base for prs to work

**Comment by chrisnojima** [comment]
@chromakode can you help me with the derivation of the merge-base stuff

**Comment by chrisnojima** [comment]
@jzila

File: shared/jenkins_test.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    yarn install --pure-lockfile
+    cd ../shared
+    check_rc $? 'visdiff fail' 1
+}
+
+if [ $test_type == 'visdiff' ]; then
+    visdiff
+elif [ $test_type == 'visdiff-install' ]; then
+    visdiff_install
+else
```
Comment (jzila) [inline shared/jenkins_test.sh]: I'd rather have an explicit `js` condition, but not a big deal either way.

**Comment by jzila** [comment]
As I mentioned in slack, I just realized that `env.CHANGE_TARGET` won't be set for master builds, so that case needs to be handled.

**Comment by chrisnojima** [comment]
closed and moved to https://github.com/keybase/client/pull/5949

**Pull request closed without merging**